e2e_deployment_files/long_haul_deployment.template.windows.json

```diff
@@ -21,14 +21,14 @@
           "edgeAgent": {
             "type": "docker",
             "settings": {
-              "image": "<Container_Registry>/microsoft/azureiotedge-agent:<Build.BuildNumber>-linux-<Architecture>",
+              "image": "<Container_Registry>/microsoft/azureiotedge-agent:<Build.BuildNumber>-windows-<Architecture>",
```

Comment:
thanks.

Reply:
I'm not sure, it always seemed to pass... Looks to be using the right images now. I'll let Slavi know
Long haul: https://msazure.visualstudio.com/One/_build/results?buildId=22834000
Stress: https://msazure.visualstudio.com/One/_build/results?buildId=22834593